backend/src/main.cpp

```diff
@@ -258,6 +258,29 @@ void SetCid(const Napi::CallbackInfo& info)
     UserSession::cid = cid;
 }
 
+void SetRadioEffects(const Napi::CallbackInfo& info)
+{
+    auto radioEffects = info[0].As<Napi::String>().Utf8Value();
```

Comment:
Might make sense to explicitly force the incoming value to lowercase so there aren't odd cases in the future where somehow someone accidentally sends "Off" or "ON" and it winds up not working. Those are irritating bugs to chase down 😂

Reply:
Done.